<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 127);
background-color: rgb(130, 249, 255);
background-color: rgb(255, 255, 255);
background-color: rgb(247, 247, 247);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Boton_carga">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Señal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_guarda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Señal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Frecuencias de interes (Hz)</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graficador">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);
background-color: rgb(249, 249, 249);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>                       Sistema básico para la visualización de señales y analisis espectral</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graficador_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 249, 249);</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Ingresar campo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Clave">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Transformada de Wavelet continua </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_generar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_original">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_welch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Arial Narrow&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 75 italic 10pt &quot;Arial Narrow&quot;;</string>
    </property>
    <property name="text">
     <string>metodo Welch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton_Multi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Arial Narrow&quot;;
font: 75 italic 10pt &quot;Arial Narrow&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 75 italic 10pt &quot;Arial Narrow&quot;;</string>
    </property>
    <property name="text">
     <string>metodo Multitaper</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Boxmenor">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Boxmayor">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>12000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>de:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Frecuencia de muestreo (Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Tamaño de la ventana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Solapamiento</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="muestreo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tamano_ventana">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="solapamiento">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Rango de frecuencia (fpass):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Factor de suavizado(Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>Tamaño de ventana (seg)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rango_frec1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="factor_suav">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tamano2_ventana">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="p">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rango_frec2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Sitka Heading&quot;;</string>
    </property>
    <property name="text">
     <string>hasta</string>
    </property>
   </widget>
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>29</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <zorder>label_15</zorder>
   <zorder>label_9</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>Boton_carga</zorder>
   <zorder>Boton_guarda</zorder>
   <zorder>label</zorder>
   <zorder>graficador</zorder>
   <zorder>label_6</zorder>
   <zorder>graficador_2</zorder>
   <zorder>line</zorder>
   <zorder>label_7</zorder>
   <zorder>Clave</zorder>
   <zorder>label_8</zorder>
   <zorder>Boton_generar</zorder>
   <zorder>Boton_original</zorder>
   <zorder>Boton_welch</zorder>
   <zorder>Boton_Multi</zorder>
   <zorder>Boxmenor</zorder>
   <zorder>Boxmayor</zorder>
   <zorder>muestreo</zorder>
   <zorder>tamano_ventana</zorder>
   <zorder>solapamiento</zorder>
   <zorder>line_2</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>rango_frec1</zorder>
   <zorder>factor_suav</zorder>
   <zorder>tamano2_ventana</zorder>
   <zorder>label_14</zorder>
   <zorder>p</zorder>
   <zorder>rango_frec2</zorder>
   <zorder>campo_grafico</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
